torch/jit/frontend.py

```diff
@@ -156,7 +156,12 @@ def _forward(self):
             In this case, the `__name__` attribute of the function object is "_forward",
             but we want the result AST to have the name "forward".
         self_name: If this function is a method, what the type name of `self` is.
+        unused: If True, this function has been decorated with @torch.jit.unused and should not be compiled.
+                Its body will be replaced with one that prints an informative error message.
     """
+    def unused_fn(*args, **kwargs):
```

Comment:
I don't think this is used at all.

Reply:
Yep, I tried to initially replace the entire input AST with the AST of this parsed function, but that wouldn't have worked because I don't think we support `*args` and `**kwargs`.